qmxgraph/page/api.js

```diff
@@ -597,6 +597,40 @@ graphs.Api.prototype.isVisible = function isVisible (cellId) {
     return model.isVisible(cell);
 };
 
+/**
+ * Sets a cell connectable property.
+ *
+ * @param {number} cellId Id of a cell in graph.
+ * @param {boolean} enabled Enable value of this property.
+ */
+graphs.Api.prototype.setConnectable = function setConnectable(cellId, enabled) {
+    "use strict";
+
+    var graph = this._graphEditor.graph;
+    var model = graph.getModel();
+    var cell = this._findCell(model, cellId);
+    if (cell) {
+        cell.setConnectable(enabled);
+    } else {
+        throw Error("Unable to find cell with id " + cellId);
+    }
+};
+
+/**
+ * Returns current cell's connectable property. If the graph has connection forbidden this will return `false`.
+ *
+ * @returns {boolean} Enable value of this property.
+ */
+graphs.Api.prototype.isConnectable = function isConnectable(cellId) {
+    "use strict";
+
+    var graph = this._graphEditor.graph;
+    var model = graph.getModel();
+    var cell = this._findCell(model, cellId);
+    return model.isConnectable(cell);
```

Comment:
This checks both cell and graph-wide `connectable` property?

Reply:
[isConnectable](https://jgraph.github.io/mxgraph/docs/js-api/files/model/mxGraphModel-js.html#mxGraphModel.isConnectable)

> Returns true if the given mxCell is connectable.  If <edgesConnectable> is false, then this function returns false for all edges else it returns the return value of mxCell.isConnectable.